packages/utils/src/packages/has-multiple-resources.ts

```diff
@@ -0,0 +1,15 @@
+import { ResourceType } from '../types/resource-type';
+
+/**
+ * Check if it is a package with multiple resources.
+ */
+export const hasMultipleResources = (resource: any, type: ResourceType) => {
+    switch (type) {
```

Comment:
Why use `switch` if there is only one case?

Reply:
Good question, IIRC we use to have more options hehehe.